### DISCUSSION TITLE
How to handle click events with this library?

### DISCUSSION THREAD

**Discussion by nhouser9** [DerYeger/yeger#242] [Q&A] [answered]
When a node in my graph is clicked I want to do something. How can I achieve that with this d3-graph-controller library? 

Sorry if it's a beginner question, I've been playing around with it for a few hours and can't find anything. I see the option to configure double click behavior, but not regular click, which is odd.

**Comment by DerYeger** [comment, marked as answer]
Single clicks can't be configured for `d3-graph-controller`, because they are already bound to the dragging feature.

**Reply by nhouser9** [reply]
Thanks for the reply. I'll use another library then, because I need this behavior
